doc/parse/package.go

```diff
@@ -33,6 +33,12 @@ func NewPackageDoc(dir string) (*doc.Package, error) {
 
 func setDocFuncsMap(pkgDoc *doc.Package) {
 	funcsMap = make(map[string]*doc.Func, len(pkgDoc.Funcs))
+	for _, tp := range pkgDoc.Types {
+		for _, fn := range tp.Methods {
+			fn.Doc = strings.Replace(fn.Doc, "\t", "    ", -1)
+			funcsMap["("+fn.Recv+")"+fn.Name] = fn
+		}
+	}
```

Comment:
I tested this locally and I get the same doc generated for a method handler with and without your code change. 
Can you add an example in examples/widgets/* or foos/* to demonstrate that the change works?